packages/lb-annotation/src/core/toolOperation/ScribbleTool.ts

```diff
@@ -314,6 +327,7 @@ class ScribbleTool extends BasicToolOperation {
           ctx.restore();
         }
       });
+      this.pushUrlForHistory();
```

Comment:
what's the meaning?

Reply:
Save the url of the canvas image into History